client/a8c-for-agencies/sections/marketplace/hosting-overview/hosting-v2/standard-agency-hosting/wpcom-plan-selector/index.tsx

```diff
@@ -90,26 +90,35 @@ export default function WPCOMPlanSelector( { onSelect }: Props ) {
 			</div>
 
 			<div className="wpcom-plan-selector__card">
-				<div className="wpcom-plan-selector__details">
-					{ ownedPlans && (
-						<div className="wpcom-plan-selector__owned-plan">
-							{ translate( 'You own %(count)s site', 'You own %(count)s sites', {
-								args: {
-									count: ownedPlans,
-								},
-								count: ownedPlans,
-								comment: '%(count)s is the number of WordPress.com sites owned by the user',
-							} ) }
+				{ ! isLicenseCountsReady && (
+					<div className="wpcom-plan-selector__details is-placeholder">
+						{ ownedPlans && <div className="wpcom-plan-selector__owned-plan"></div> }
+						<div className="wpcom-plan-selector__plan-name"></div>
+						<div className="wpcom-plan-selector__price"></div>
+						<div className="wpcom-plan-selector__price-interval"></div>
+						<div className="wpcom-plan-selector__cta">
+							<div className="wpcom-plan-selector__cta-label"></div>
+							<div className="wpcom-plan-selector__cta-component"></div>
 						</div>
-					) }
+					</div>
+				) }
 
-					<h2 className="wpcom-plan-selector__plan-name">{ planName }</h2>
+				{ isLicenseCountsReady && (
+					<div className="wpcom-plan-selector__details">
+						{ ownedPlans && (
+							<div className="wpcom-plan-selector__owned-plan">
+								{ translate( 'You own %(count)s site', 'You own %(count)s sites', {
+									args: {
+										count: ownedPlans,
+									},
+									count: ownedPlans,
+									comment: '%(count)s is the number of WordPress.com sites owned by the user',
+								} ) }
+							</div>
+						) }
```

Comment:
[NITPICK]

I think we could do 

`isLicenseCountsReady ? <Render1 /> : <Render2 />`

Reply:
Good suggestion. I have applied it 👍 https://github.com/Automattic/wp-calypso/pull/93099/commits/1ec952696f2a95651aa53c5126bfa7b9efed1725